src/main/resources/migrations/00072_remove_event_sqs_message_id_null_constraint.sql

```diff
@@ -0,0 +1,4 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:remove_event_sqs_message_id_not_null_constraint
+ALTER TABLE event ALTER COLUMN sqs_message_id DROP NOT NULL;
```

Comment:
This has sneaked in from the other PR!

Reply:
It will go when rebased! Just borrows the commit for integration tests